packages/googlecloud/functions/getanswer/inquirer.py

```diff
@@ -378,8 +384,9 @@ def answer_query(
     db_pdf: any,
     db_pc: any,
     db_news: any,
+    return_context: any = False,
```

Comment:
default value of return_context is False - so if not being called it should run exactly how it did before and just return one object instead of two